opendut-util/opendut-auth/src/confidential/blocking_client.rs

```diff
@@ -0,0 +1,226 @@
+use std::fmt::Formatter;
+use std::sync::Arc;
+use std::time::Duration;
+use chrono::{NaiveDateTime, Utc};
+use config::Config;
+use oauth2::{AccessToken, TokenResponse};
+use oauth2::basic::{BasicClient, BasicTokenResponse};
+use std::sync::{RwLock, RwLockWriteGuard};
+use tokio::task::JoinError;
+use tracing::debug;
+use crate::confidential::config::{ConfidentialClientConfig, ConfidentialClientConfigData};
+use crate::confidential::blocking_reqwest_client::OidcBlockingReqwestClient;
+use crate::confidential::error::{ConfidentialClientError, WrappedRequestTokenError};
+
+#[derive(Debug)]
+pub struct ConfidentialClient {
```

Comment:
This client is new. Move to own `blocking` submodule: opendut_auth::confidential::blocking